pkg/booleanpolicy/querybuilders/special_cases.go

```diff
@@ -29,12 +29,32 @@ func ForK8sRBAC() QueryBuilder {
 // so we want to find deployments that don't drop these capabilities.
```

Comment:
can you update the comment to also mention the implicit ALL case?

Reply:
Done